__tests__/app.test.js

```diff
@@ -449,3 +449,30 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe('DELETE /api/comments/:comment_id', () => {
+  test('Status: 204, deletes the comment', () => {
+    return request(app)
+    .delete('/api/comments/2')
+    .expect(204)
+    .then(({body}) => {
+      expect(body).toEqual({});
+    });
+  });
```

Comment:
You dont need this bit as it doesn't return anything. 
If you really wanted, you could add a query to the db to check its gone, in here, but up to you